frontend/src/app/scratch/[slug]/opengraph-image.tsx

```diff
@@ -0,0 +1,123 @@
+                    </div>
+                </div>
+                <div tw="flex mt-3 ml-15 mr-15 text-slate-300">{preset?.name || "Custom Preset"}</div>
+            </div>
+
+            <div tw="flex justify-between mt-5 ml-15 mr-15">
+                <div tw="flex">
+                    {score === -1
+                        ?
+                        <div tw="flex">
+                            <div tw="flex flex-col justify-around">
+                                <XCircleFillIcon width={48} height={48} />
+                            </div>
+                            <div tw="flex flex-col justify-around ml-5">
+                                <div tw="flex">No Score</div>
+                            </div>
+                        </div>
+                        : score === 0 || scratch.match_override
+                            ?
+                            <div tw="flex">
+                                <div tw="flex flex-col justify-around">
+                                    <CheckCircleFillIcon width={48} height={48}/>
+                                </div>
+                                <div tw="flex flex-col items-center ml-5">
+                                    <div tw="flex">Matched</div>
+                                    {scratch.match_override &&
+                                        <div tw="flex text-4xl text-slate-300">(Override)</div>
+                                    }
+                                </div>
+                            </div>
+                            :
+                            <div tw="flex">
+                                <div tw="flex flex-col justify-around">
+                                    <TrophyIcon width={48} height={48} />
+                                </div>
+                                <div tw="flex flex-col justify-around ml-5">
+                                    <div tw="flex">
+                                        {score} ({percentToString(percent)})
+                                    </div>
+                                </div>
+                            </div>
+                    }
+
+                    {parentScratch &&
+                    <div tw="flex ml-10">
+                        <div tw="flex flex-col justify-around">
+                            <RepoForkedIcon width={48} height={48} />
+                        </div>
+                        <div tw="flex flex-col justify-around ml-5">
+                            {parentScratch.owner?.username ?? "Anonymous User"}
+                        </div>
+                    </div>
+                    }
+                </div>
+
+                <div tw="flex flex-col justify-around ml-15">
+                    <PurpleFrog width={64} height={64} />
+                </div>
+
+            </div>
+            <div tw="flex">
+                <div tw={`flex h-4 bg-purple-500 w-${doneWidth}`}></div>
+                <div tw={`flex h-4 bg-purple-900 w-${todoWidth}`}></div>
```

Comment:
`w-300` in Tailwind doesn't correspond to 300 pixels, it instead corresponds to a special scale: https://tailwindcss.com/docs/width

Instead, these should be [arbitrary values](https://tailwindcss.com/docs/width#arbitrary-values):
```suggestion
                <div tw={`flex h-4 bg-purple-500 w-[${doneWidth}`}px]></div>
                <div tw={`flex h-4 bg-purple-900 w-[${todoWidth}`}px]></div>
```

Reply:
Makes sense, thanks for all the feedback, have pushed a fresh commit!